data/README.md

```diff
@@ -39,18 +39,27 @@ instance, they should be in the same format as the files below.
   Contains signature (node) activity levels (as absolute values) for each
   sample in the compendium derived from the eADAGE machine learning model.
   In this file, samples are rows, and signatures are columns. This file is
-  based on the
-  `all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt`
-  file, which is now in the `data/old/` folder and was generated by
+  based on:
+  ```
+  all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt
+  ```
+
+  which is now in the `data/old/` folder and was generated by
   [this script](https://bitbucket.org/greenelab/eadage/src/tip/node_interpretation/write_HWactivity.R).
 
+  **Note:** The signatures shown on the web (click
+  [here](http://adage.greenelab.com/#/signature/search?mlmodel=1) to see a
+  demo) will be in **alphabetical order**. If you would like them to be ranked
```

Comment:
Saying "ranked" is technically correct, but I think "sorted" is a more common way to put this and would sound more natural.

Reply:
Hmm, originally I thought `sorted` sounds a little too "techie". I will change it to "sorted" then.

Reply:
Hah! I had the opposite impression. Either way is fine with me... it's a minor thing.

Reply:
Instead of "sorted" or "ranked", maybe we should use the more generic word "displayed".

Reply:
"If you would like them ordered differently..." is another option.